libs/core/models/auth/registration.model.ts

```diff
@@ -0,0 +1,18 @@
+/** Model for registration. */
+export interface RegistrationModel {
```

Comment:
The postfix is redundant here. It's already clear that it's a model
```suggestion
export interface Registration {
```